data/hlint.yaml

```diff
@@ -823,6 +823,17 @@
     - warn: {lhs: either f g (Right y), rhs: g y, name: Evaluate}
     - warn: {lhs: "fst (x,y)", rhs: x, name: Evaluate}
     - warn: {lhs: "snd (x,y)", rhs: "y", name: Evaluate}
+    - warn: {lhs: fromJust (Just x), rhs: x, name: Evaluate}
+    - warn: {lhs: fromJust . Just, rhs: id, name: Evaluate}
```

Comment:
You don't need to give this version, since hlint will infer the point free version